net-libs/canlock/canlock-3.3.1.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="A library for creating and verifying Usenet cancel locks"
+HOMEPAGE="https://micha.freeshell.org/libcanlock/"
+SRC_URI="https://micha.freeshell.org/lib${PN}/src/lib${P}.tar.bz2"
+S="${WORKDIR}/lib${P}"
+
+LICENSE="BSD MIT"
```

Comment:
The MIT-like license has been clarified as being ICU.  They added a SPDX-License-Identifier to every file.